server/models/default_local_provider.go

```diff
@@ -33,6 +34,7 @@ import (
 	"github.com/spf13/viper"
 	"gopkg.in/yaml.v2"
 	"gorm.io/gorm"
+	"k8s.io/client-go/util/homedir"
```

Comment:
What is this for?

Reply:
https://github.com/meshery/meshery/pull/11904/files#diff-0e12afadeea6ac8911508be98ace80904e870a10b2c83c2abe6472b4e6198386R271-R273
For this it was used in the remote provider so I used this.